### PR TITLE
PDA-47 Process Diagram Endpoint

### PR DESCRIPTION
**Pull request by ffleandro**

### CODE REVIEW

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by entando-service** [comment]
Something went wrong. Investigate!

File: src/main/java/org/entando/plugins/pda/controller/ProcessController.java [hunk 1]
```diff
@@ -42,4 +44,17 @@
...
     }
+
+    @Secured(PROCESS_DIAGRAM)
+    @ApiOperation(notes = "Gets the process diagram", nickname = "getProcessDiagram",
+            value = "GET Process Diagram")
+    @GetMapping(path = "/{id}/diagram", produces = {"application/svg+xml"})
+    public byte[] getProcessDiagram(@PathVariable final String connId,
+            @PathVariable final String id) {
+        log.info("Retrieving process diagram for {}", id);
+        Connection connection = connectionService.get(connId);// NOPMD
```
Comment (werbth) [inline src/main/java/org/entando/plugins/pda/controller/ProcessController.java]: I think this `NOPMD` comment here is not required anymore. Please, check

**Review by werbth — DISMISSED** [review]

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by werbth** [comment]
Retest this please

**Comment by entando-service** [comment]
Something went wrong. Investigate!

**Comment by werbth** [comment]
Retest this please

**Comment by entando-service** [comment]
All is well

**Review by werbth — APPROVED** [review]

**Pull request merged**